api/src/main/java/org/pmiops/workbench/api/WorkspaceController.java

```diff
@@ -100,20 +113,39 @@ private void setCdrVersionId(Workspace workspace,
     }
   }
 
+  private FirecloudWorkspaceId generateFirecloudWorkspaceId(String name) {
```

Comment:
OK, I'm un-confused (thanks Dan!). For now, we'll maintain 1:1 correspondance between project and workspace, and simply keep a single billing account associated with all projects.